bayesian/v2/api_v2.py

```diff
@@ -15,29 +15,37 @@
 #
 """Definition of all v2 REST API endpoints of the server module."""
 
+import os
 import urllib
 import time
+
 from requests_futures.sessions import FuturesSession
 from flask import Blueprint, current_app, request, g
 from flask.json import jsonify
 from flask_restful import Api, Resource
-from f8a_worker.utils import (MavenCoordinates, case_sensitivity_transform)
-from fabric8a_auth.auth import login_required
+
+from bayesian.v2.utility import VendorAnalyses
+from bayesian.v2.stack_analyses import StackAnalyses
+
```

Comment:
Any particular reason for extra line breaks..?

Reply:
Just grouped all similar imports together,